templates/media/media-with-message.php

```diff
@@ -0,0 +1,155 @@
+						}
+					jQuery( '.new-message').hide().slideDown( 200, function() {
+						jQuery( '.new-message' ).removeClass( 'new-message' );
+					});
+					}
+					jQuery( button ).removeClass( 'loading' );
+				});
+				return false;
+			};
+			$( 'input#send_reply_button' ).unbind( 'click' ).bind('click', handler);
+		});
+
+	</script>
+	<span class="primary rtm-media-msg-upload-button rtmedia-upload-media-link" 
+		id="rtm_show_upload_ui" title="Upload Media">
+		<i class="dashicons dashicons-upload rtmicon"></i>
+			Upload Media File
+	</span>
+	<div id="rtm-media-gallery-uploader" class="rtm-media-gallery-uploader">
+		<?php
+		rtmedia_uploader(
+			[
+				'is_up_shortcode' => false,
+				'allow_anonymous' => true,
+				'privacy_enabled' => false,
+			]
+		);
+			?>
+	</div>
+	<input type="hidden" id="rtm_bpm_uploaded_media" name="rtm_bpm_uploaded_media" />
+	<?php
+}
+
+/**
+ * Add additional parameter media ID in BuddyPress message SEND MESSAGE process.
+ * And Insert data into MEDIA_META table.
+ *
+ * @param object $message Getting parameter response when sending message.
+ **/
+function rtm_add_message_media_params( $message ) {
+	$insert_media_object  = new RTDBModel( 'rtm_media_meta' );
+	$message->media_array = filter_input( INPUT_POST, 'rtm_bpm_uploaded_media' );
+	$media                = explode( ',', $message->media_array );
+	if ( ! empty( $media ) && null !== $media ) {
+		foreach ( $media as $media_id ) {
+			$insert_media_object->insert(
+				[
+					'media_id'   => $media_id,
+					'meta_key'   => 'rtm-bp-message-media', // phpcs:ignore
+					'meta_value' => $message->id, // phpcs:ignore
+				]
+			);
+		}
+	}
+	?>
+	<script>
+		jQuery( '#msg-success-bp-msg-media' ).hide();
+		jQuery( '.rtm-media-msg-upload-button' ).attr( 'id', 'rtm_show_upload_ui' );
+		jQuery( '.rtm-media-msg-upload-button' ).html( '' );
+		jQuery( '.rtm-media-msg-upload-button' ).html( jQuery( '<i>', { class: 'dashicons dashicons-upload rtmicon' }{ text: 'Upload Media File' } ) );
+	</script>
+	<?php
+}
+
+/**
+ * As a result show attached media with message.
+ **/
+function show_rtm_bp_msg_media() {
+	$get_data_object = new RTDBModel( 'rtm_media_meta' );
+	$media_result    = $get_data_object->get( [ 'meta_value' => bp_get_the_thread_message_id() ] );  // phpcs:ignore
+	$url             = explode( 'messages/', sanitize_text_field( wp_unslash( filter_input( INPUT_SERVER, 'REQUEST_URI' ) ) ) );
+	if ( '0' !== $media_result[0]->media_id ) {
+	?>
+	<ul class='rtmedia-list-media rtm-gallery-list clearfix' style = 'margin-top: 10px;'>
+	<?php
+	foreach ( $media_result as $media_result_array_value ) {
+		$media     = rtmedia_image( 'rt_media_thumbnail', $media_result_array_value->media_id, false );
+		$media_url = $url[0] . '/media/' . $media_result_array_value->media_id . '/';
+		?>
+
+			<li class='rtmedia-list-item' style='display:inline; float: left;'' id="<?php echo $media_result_array_value->media_id; // @codingStandardsIgnoreLine?>">
```

Comment:
don't use inline css if possible 
use `esc_attr` for id
`style='display:inline; float: left;''` ?